### PR TITLE
Fix/comments and typos

### PR DESCRIPTION
**Pull request by ericlp**

### CODE REVIEW

File: core/src/main/se/tevej/game/view/EntityRenderable.java [hunk 1]
```diff
@@ -3,6 +3,10 @@
 import com.badlogic.ashley.core.Entity;
 import main.se.tevej.game.view.rendering.TBatchRenderer;
 
+/**
+ *  An EntityRenderable is a way of rendering an certain entity.
```
Comment (jerge) [inline core/src/main/se/tevej/game/view/EntityRenderable.java]: an -> a

File: core/src/main/se/tevej/game/view/View.java [hunk 2]
```diff
@@ -66,7 +66,7 @@ public void render(){
 
     /**
      * Goes through all the components of an added entity and checks if it has a EntityRenderable for it.
```
Comment (jerge) [inline core/src/main/se/tevej/game/view/View.java]: a -> an

**Comment by ViddeM** [comment]
Fix conflicts and I can approve.

File: core/src/main/se/tevej/game/libgdx/view/rendering/input/KeyBoardLibgdxAdapter.java [hunk 3]
```diff
@@ -14,6 +14,10 @@
 
     static final Map<Integer, TButton> inputMap = new HashMap<>();
 
+    /**
```
Comment (Portals) [inline core/src/main/se/tevej/game/libgdx/view/rendering/input/KeyBoardLibgdxAdapter.java]: We should not explain a static block. The comment should talk about the mapping between TKey and libGDXs keycodes

**Pull request closed without merging**